utils/pckgen

```diff
@@ -151,28 +152,28 @@ check_version() {
     fi
 
     if [ -d "$SOURCES_DIR/.git" ]; then
-        CAS_VERSION_BUILD=$(printf %04d $(git -C "$SOURCES_DIR" log --merges --oneline | wc -l))
-        LAST_COMMIT_HASH=$(git -C "$SOURCES_DIR" log -1 --pretty=format:%H)
-        LAST_COMMIT_HASH_ABBR=$(git -C "$SOURCES_DIR" log -1 --pretty=format:%h)
-        LAST_COMMIT_DATE=$(git -C "$SOURCES_DIR" log -1 --pretty=format:%ci |\
+        CAS_VERSION_BUILD=$(printf %04d $(cd "$SOURCES_DIR" && git log --merges --oneline | wc -l))
+        LAST_COMMIT_HASH=$(cd "$SOURCES_DIR" && git log -1 --pretty=format:%H)
+        LAST_COMMIT_HASH_ABBR=$(cd "$SOURCES_DIR" && git log -1 --pretty=format:%h)
+        LAST_COMMIT_DATE=$(cd "$SOURCES_DIR" && git log -1 --pretty=format:%ci |\
                            sed "s/ /T/" | sed "s/ //" | sed "s/00$/:00/")
-        LAST_COMMIT_TIMESTAMP=$(git -C "$SOURCES_DIR" log -1 --pretty=format:%ct)
+        LAST_COMMIT_TIMESTAMP=$(cd "$SOURCES_DIR" && git log -1 --pretty=format:%ct)
         for SUBMOD in ${SUBMODULES[@]}; do
-            LAST_SUB_COMMIT_HASHES+=($(git -C "$SOURCES_DIR/$SUBMOD" log -1 --pretty=format:%H))
-            LAST_SUB_COMMIT_HASHES_ABBR+=($(git -C "$SOURCES_DIR/$SUBMOD" log -1 --pretty=format:%h))
+            LAST_SUB_COMMIT_HASHES+=($(cd "$SOURCES_DIR/$SUBMOD" && git log -1 --pretty=format:%H))
+            LAST_SUB_COMMIT_HASHES_ABBR+=($(cd "$SOURCES_DIR/$SUBMOD" && git log -1 --pretty=format:%h))
         done
     elif [ -f "$VERSION_FILE" ]; then
-        source "$VERSION_FILE" &> /dev/null
+        source "$VERSION_FILE" &>/dev/null
     else
         error "Couldn't obtain CAS version - no git tree nor version file within given sources"
     fi
 
     VERSION="${CAS_VERSION_MAIN}.${CAS_VERSION_MAJOR}.${CAS_VERSION_MINOR}.${CAS_VERSION_BUILD}"
 
-    if [ $(git -C "$SOURCES_DIR" tag --points-at HEAD) ]; then
+    if [ $(cd "$SOURCES_DIR" && git tag --points-at HEAD) ]; then
         RELEASE="release"
-    elif [ $(git -C "$SOURCES_DIR" log -1 --pretty=format:%H)\
-           == $(git -C "$SOURCES_DIR" log -1 --merges --pretty=format:%H) ]; then
+    elif [ $(cd "$SOURCES_DIR" && git log -1 --pretty=format:%H)\
+           == $(cd "$SOURCES_DIR" && git log -1 --merges --pretty=format:%H) ]; then
         RELEASE="master"
     else
         RELEASE="devel.${LAST_COMMIT_TIMESTAMP}"
```

Comment:
This info should also be part of version file. In case there is no git repository (e.g. sources were extracted from archive) we will always fallback to "devel", which in most cases will be wrong, as zip/tar repositories will mostly be used in case of release versions.